backend/src/main/java/com/beaplab/BeaplabEngine/controller/LoginUserController.java

```diff
@@ -116,19 +119,26 @@ public ResponseEntity<String> checkAuth(HttpServletRequest request) {
         Iterator<? extends ExpiringSession> iterator = usersSessions.iterator();
         while (iterator.hasNext()) {
             Set<String> set = iterator.next().getAttributeNames();
-            for (String str: set)
+            for (String str: set){
                 logger.info("str: " + str);
+                rollbar.info("str: " + str);
+            }
+
         }
 
         if (!usersSessions.isEmpty()){
             logger.info("A valid session exists for username: " + username);
+            rollbar.info("A valid session exists for username: " + username)
```

Comment:
This is making your build fail you are missing the `;`
```suggestion
            rollbar.info("A valid session exists for username: " + username);
```